master/internal/rm/kubernetesrm/pods.go

```diff
@@ -935,12 +935,20 @@ func (p *pods) summarize(ctx *actor.Context) (map[string]model.AgentSummary, err
 		quotaList, err := p.quotaInterfaces[namespace].List(context.TODO(), metaV1.ListOptions{})
 		if err != nil && !k8serrors.IsNotFound(err) {
 			return nil, err
-		} else if k8serrors.IsNotFound(err) || quotaList == nil || len(quotaList.Items) != 1 {
+		} else if k8serrors.IsNotFound(err) || quotaList == nil {
+			continue
+		}
+
+		relevantQuotas := cpuAndGpuQuotas(quotaList)
+		if len(relevantQuotas) != 1 {
 			// TODO: figure out how we want to handle multiple quotas per namespace?
+			// When there's multiple conflicting quotas, k8s seems to use the most
+			// restrictive of them—i.e. if there's a quota limiting to 100 CPUs and one
+			// limiting to 10, only 10 CPUs will be allowed.
 			continue
```

Comment:
nit: can cut one line by using `len(relevantQuotas) == 1` and putting relevant logic inside. Not blocking.

Reply:
The Go idiom in general is to handle edge cases using things like `if`s and putting the happy path ad de-indented as possible. I prefer this style, even at the cost of extra lines.